services/keyvault/auth/auth.go

```diff
@@ -0,0 +1,37 @@
+package auth
+
+import (
+	"os"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+)
+
+// NewAuthorizerFromEnvironment creates a keyvault dataplane Authorizer configured from environment variables in the order:
+// 1. Client credentials
+// 2. Client certificate
+// 3. Username password
+// 4. MSI
+func NewAuthorizerFromEnvironment() (autorest.Authorizer, error) {
```

Comment:
changelog update and sdk version update ?

Reply:
Updated